package.json

```diff
@@ -1,10 +1,14 @@
 {
-  "name": "Skizzle",
-  "version": "3.4.0-beta",
+  "name": "skizzle",
+  "version": "0.0.0",
```

Comment:
1.0.0-beta.1 ?

Reply:
i don't know, the expected result is to automate the package versionning with changelog